src/App.jsx

```diff
@@ -1,107 +1,39 @@
-          </h3>
-
-          <p>
-            {' Posted by  '}
-
-            <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-              Patricia Lebsack
-            </a>
-          </p>
-        </div>
-
-        <p className="PostInfo__body">
-          deserunt eos nobis asperiores et hic est debitis repellat molestiae
-          optio nihil ratione ut eos beatae quibusdam distinctio maiores earum
-          voluptates et aut adipisci ea maiores voluptas maxime
-        </p>
-
-        <div className="CommentList">
-          <div className="CommentInfo">
-            <div className="CommentInfo__title">
-              <strong className="CommentInfo__name">pariatur omnis in</strong>
-
-              {' by '}
-
-              <a
-                className="CommentInfo__email"
-                href="mailto:Telly_Lynch@karl.co.uk"
-              >
-                Telly_Lynch@karl.co.uk
-              </a>
-            </div>
-
-            <div className="CommentInfo__body">
-              dolorum voluptas laboriosam quisquam ab totam beatae et aut
-              aliquid optio assumenda voluptas velit itaque quidem voluptatem
-              tempore cupiditate in itaque sit molestiae minus dolores magni
-            </div>
-          </div>
-
-          <div className="CommentInfo">
-            <div className="CommentInfo__title">
-              <strong className="CommentInfo__name">
-                odio adipisci rerum aut animi
-              </strong>
-
-              {' by '}
-
-              <a
-                className="CommentInfo__email"
-                href="mailto:Nikita@garfield.biz"
-              >
-                Nikita@garfield.biz
-              </a>
-            </div>
-
-            <div className="CommentInfo__body">
-              quia molestiae reprehenderit quasi aspernatur aut expedita
-              occaecati aliquam eveniet laudantium omnis quibusdam delectus
-              saepe quia accusamus maiores nam est cum et ducimus et vero
-              voluptates excepturi deleniti ratione
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </section>
-);
+import { PostList } from './components/PostList';
+
+export const App = () => {
+  const postsFromServer = [
+    {
+      id: 1,
+      title: 'qui est esse',
+      body: 'est rerum tempore vitae sequi sint nihil reprehenderit...',
+      userId: 1,
+      user: {
+        id: 1,
+        name: 'Leanne Graham',
+        email: 'Sincere@april.biz',
+      },
+      comments: [
+        {
+          id: 1,
+          name: 'pariatur omnis in',
+          email: 'Telly_Lynch@karl.co.uk',
+          body: 'dolorum voluptas laboriosam quisquam ab totam beatae...',
+        },
+        {
+          id: 2,
+          name: 'odio adipisci rerum aut animi',
+          email: 'Nikita@garfield.biz',
+          body: 'quia molestiae reprehenderit quasi aspernatur aut expedita...',
+        },
+      ],
+    },
+  ];
```

Comment:
you should import these posts, and add users, and comments there

Reply:
how? I don't quite understand.

Reply:
here were an example at the forked repo
```
// import postsFromServer from './api/posts.json';
// import commentsFromServer from './api/comments.json';
// import usersFromServer from './api/users.json';
```

Reply:
so we will see not only one user
<img width="1753" alt="Screenshot 2023-12-09 at 23 07 57" src="https://github.com/mate-academy/react_static-list-of-posts-js/assets/117587463/d5969aaf-e8ef-4053-bddb-10370328f08f">